src/common/unix_timestamp.rs

```diff
@@ -0,0 +1,13 @@
+use chrono::{DateTime, Utc};
+
+
+pub trait UnixTimestamp {
+    fn ts2date(ts: i64) -> String;
+}
```

Comment:
I'd prefer something like the following
```suggestion
pub trait ForensicsTimestamp {
    fn ts2date(&self) -> String;
}
```